README.md

```diff
@@ -1,17 +1,77 @@
 
 ## Contributors ##
 
 NUnit 3.0 was created by [Charlie Poole](https://github.com/CharliePoole), [Rob Prouse](https://github.com/rprouse), [Simone Busoli](https://github.com/simoneb), [Neil Colvin](https://github.con/oznetmaster) and numerous community contributors. A complete list of contributors since NUnit migrated to GitHub can be [found on GitHub](https://github.com/nunit/nunit/graphs/contributors).
 
 Earlier versions of NUnit were developed by Charlie Poole, James W. Newkirk, Alexei A. Vorontsov, Michael C. Two and Philip A. Craig.
+
+
+## NUnit Projects ##
+
+NUnit is made up of several projects. When reporting issues, please try to report issues in the correct project.
+
+### Core Projects ###
+
+- [NUnit Test Framework](https://github.com/nunit/nunit) - The test framework used to write NUnit tests (this repository)
+- [NUnit Console and Engine](https://github.com/nunit/nunit-console) - Runs unit tests from the command line and provides the engine that is used by other test runners to run NUnit tests
+
+### Visual Studio Extensions ###
+
+- [NUnit 3 Visual Studio Adapater](https://github.com/nunit/nunit3-vs-adapter) - Visual Studio adapter for running NUnit 3 tests in Visual Studio and in VSTS/TFS builds
+- [NUnit Visual Studio Templates](https://github.com/nunit/nunit-vs-templates) - Project templates and snippets for writing unit tests in Visual Studio
+- [Visual Studio Test Generator](https://github.com/nunit/nunit-vs-testgenerator) - Generates NUnit tests in Visual Studio
+
+### Other Projects ###
+
+- [NUnit GUI](https://github.com/nunit/nunit-gui) - Runs NUnit 3 tests in a GUI based runner
+- [NUnit Xamarin Runner](https://github.com/nunit/nunit.xamarin) - Runs NUnit 3 tests on mobile devices using the Xamarin framework
+- [NUnit .NET Core Runner](https://github.com/nunit/dotnet-test-nunit) - Runs NUnit tests compiled against .NET Core
+- [NUnit Installer](https://github.com/nunit/nunit-distribution) - Creates the installer for the NUnit Console, Engine and Extensions
+- [NUnit Project Editor](https://github.com/nunit/nunit-project-editor) - Edits the NUnit Project format
+
+### NUnit Engine Extensions ###
+
+- [NUnit 2 Driver](https://github.com/nunit/nunit-v2-framework-driver) - Allows NUnit 3 to run NUnit 2 tests
+- [NUnit 2 Result Writer](https://github.com/nunit/nunit-v2-result-writer) - Writes test results in the legacy NUnit 2 format
+- [Visual Studio Project Loader](https://github.com/nunit/vs-project-loader) - Loads and parses Visual Studio projects and solutions
+- [TeamCity Event Listenter](https://github.com/nunit/teamcity-event-listener) - Allows you to run and integrate NUnit tests into TeamCity
+
+### Samples ###
+
+- [C# NUnit Test Samples](https://github.com/nunit/nunit-csharp-samples)
+- [Visual Basic NUnit Test Samples](https://github.com/nunit/nunit-vb-samples)
+- [F# NUnit Test Samples](https://github.com/nunit/nunit-fsharp-samples)
+
```

Comment:
This is missing C++. I wonder if we actually want these and the 'older projects' section - I guess it depends how definitive a list we want this to be. An alternative suggestion (or possibly aswell), how about adding a link to https://github.com/nunit/docs/wiki/Samples in the documentation section above? Thought process: code samples are good 'getting started' content.